src/constants/routes.js

```diff
@@ -1,5 +1,5 @@
 const ROUTES = {
-  login: '/login',
+  base: '/',
   signUp: '/sign-up'
```

Comment:
i would keep a route for login

Reply:
And the home route `/` should be a redirect to `/books` or the custom view you want